src/pages/patientView/genomicOverview/Tracks.tsx

```diff
@@ -14,6 +14,8 @@ interface TracksPropTypes {
     width:number;
 }
 
+export const DEFAULT_CHROMOSOME="GRCh37";
```

Comment:
just want to confirm this belongs in `Tracks`, so it is a `Tracks`-specific constant that's not used elsewhere. otherwise, it should go in a utils file that's more generally used.